build-logic/plugins/src/main/java/ch/srgssr/pillarbox/gradle/internal/ProjectExtensions.kt

```diff
@@ -26,12 +27,14 @@ internal fun Project.configureAndroidModule(extension: CommonExtension<*, *, *,
     compileOptions {
         sourceCompatibility = AppConfig.javaVersion
         targetCompatibility = AppConfig.javaVersion
+        isCoreLibraryDesugaringEnabled = true
     }
 
     buildFeatures {
         resValues = false
         shaders = false
     }
+    dependencies.add("coreLibraryDesugaring", libs.findLibrary("desugar_jdk_libs").get())
```

Comment:
Not sure if this really belongs here. `dependencies` comes from the `project`, not the Android extension.
Maybe this should be done in `PillarboxAndroidApplicationPlugin` and `PillarboxAndroidLibraryPlugin`?